pages/Transactions/index.tsx

```diff
@@ -59,86 +60,32 @@ const TransactionsPage: React.FunctionComponent = () => {
- 
-       const temp = [
-           {date: new Date(), username: 'Firstname Lastname', fid: 'H1234', admin: 'Firstname Lastname', message: 'short messaging explaining what the transaction was', change: 10},
-           {date: new Date(), username: 'Jacob Kim', fid: '431', admin: 'Cindy Zhang', message: 'Jacob > Cindy', change: -10},
-           {date: new Date(), username: 'bababooey', fid: '123', admin: 'Espinosa Dad', message: 'im not drunk i swear i am not i really am not i swera to god not drunk oh look its two lines and it looks good!', change: -10}
-       ]
- 
-       const renderHistory = () => {
- 
-           return(
-               <List className={styles['list']}>
-                   {
-                       temp.map((transaction) => {
-                           return(
-                               <TransactionItem date={transaction.date} username={transaction.username} fid={transaction.fid}
-                               admin={transaction.admin} message={transaction.message} change={transaction.change}/>
-                           )
-                       })
-                   }
-               </List>
-           )
-       }
+       
       
        return (
-           <Box sx={{ width: '100%', maxHeight: '62%'}}>
-           <Box>
-               <Tabs value={value} onChange={handleChange} TabIndicatorProps={{style: {background: "white"}}}
-               textColor={"inherit"} aria-label="basic tabs example">
-               <Tab label="History" className={value==0 ? styles['sel-tab'] : styles['tab']}/>
-               <Tab label="Redemptions" className={value==1 ? styles['sel-tab'] : styles['tab']}/>
-               <Tab label="Earnings" className={value==2 ? styles['sel-tab'] : styles['tab']}/>
-               </Tabs>
-           </Box>
+           <Box className={styles['transaction-container']}>
+               <Tabs value={value} onChange={handleChange} TabIndicatorProps={{style: {display: 'none'}}}
+                textColor={"inherit"} aria-label="basic tabs example" className={styles['tabs']}>
+                    <Tab label="History" className={value==0 ? styles['sel-tab'] : styles['tab']}/>
+                    <Tab label="Redemptions" className={value==1 ? styles['sel-tab'] : styles['tab']}/>
+                    <Tab label="Earnings" className={value==2 ? styles['sel-tab'] : styles['tab']}/>
+                </Tabs>
                <TabPanel value={value} index={0}>
                    <div>
                        {renderFilterHeader()}
-                       {renderCategoryHeader()}
-                       {renderHistory()}
+                       <TransactionList tabIndex={0}/>
                    </div>
                </TabPanel>
                <TabPanel value={value} index={1}>
                    <div>
                        {renderFilterHeader()}
-                       {renderCategoryHeader()}
-                       {renderHistory()}
+                       <TransactionList tabIndex={1}/>
```

Comment:
Hmm I think it would be better to load the data all at once and then display it. (When we switch tabs, we can see the delay in the data being loaded, we can discuss this more at work sash)

Reply:
I'll do this once you change the structure of firebase